test/fixtures/CSVerifier/generator.mjs

```diff
@@ -0,0 +1,169 @@
+import fs from "fs";
```

Comment:
Why is file extension mjs? It is not minimised

Reply:
It's to make node to think of the script as of a module for imports to work.